pages/edge/tutorials/cdn-on-wasmer-edge.mdx

```diff
@@ -0,0 +1,357 @@
+
+#### Log request Remote Address if available
+log-remote-address = false
+
+#### Redirect to trailing slash in the requested directory uri
+redirect-trailing-slash = true
+
+#### Check for existing pre-compressed files
+compression-static = true
+
+#### Health-check endpoint (GET or HEAD `/health`)
+health = false
+
+#### List of index files
+# index-files = "index.html, index.htm"
+#### Maintenance Mode
+
+maintenance-mode = false
+# maintenance-mode-status = 503
+# maintenance-mode-file = "./maintenance.html"
+
+### Windows Only
+
+#### Run the web server as a Windows Service
+# windows-service = false
+
+
+[advanced]
+
+#### HTTP Headers customization (examples only)
+
+#### a. Oneline version
+# [[advanced.headers]]
+# source = "**/*.{js,css}"
+# headers = { Access-Control-Allow-Origin = "*" }
+
+#### b. Multiline version
+# [[advanced.headers]]
+# source = "/index.html"
+# [advanced.headers.headers]
+# Cache-Control = "public, max-age=36000"
+# Content-Security-Policy = "frame-ancestors 'self'"
+# Strict-Transport-Security = "max-age=63072000; includeSubDomains; preload"
+
+#### c. Multiline version with explicit key (dotted)
+# [[advanced.headers]]
+# source = "**/*.{jpg,jpeg,png,ico,gif}"
+# headers.Strict-Transport-Security = "max-age=63072000; includeSubDomains; preload"
+
+
+### URL Redirects (examples only)
+
+# [[advanced.redirects]]
+# source = "**/*.{jpg,jpeg}"
+# destination = "/images/generic1.png"
+# kind = 301
+
+# [[advanced.redirects]]
+# source = "/index.html"
+# destination = "https://static-web-server.net"
+# kind = 302
+
+### URL Rewrites (examples only)
+
+# [[advanced.rewrites]]
+# source = "**/*.{png,ico,gif}"
+# destination = "/assets/favicon.ico"
+## Optional redirection
+# redirect = 301
+
+# [[advanced.rewrites]]
+# source = "**/*.{jpg,jpeg}"
+# destination = "/images/sws.png"
+
+### Virtual Hosting
+
+# [[advanced.virtual-hosts]]
+## But if the "Host" header matches this...
+# host = "sales.example.com"
+## ...then files will be served from here instead
+# root = "/var/sales/html"
+
+# [[advanced.virtual-hosts]]
+# host = "blog.example.com"
+# root = "/var/blog/html"
+```
+
+<Callout type="info" emoji="ℹ️">
+  The above configuration is the configuration of the `static-web-server`
+  package. More info about it can be found
+  [here](https://static-web-server.net/configuration/config-file/).
+</Callout>
+
+Now, let's add the following configuration to the `wasmer.toml` file.
+
+```yaml
+[package]
+name = "<your-namespace>/<your-cdn-name>"
+version = "0.1.0"
+description = "<your-namespace>/<your-cdn-name> CDN on Wasmer Edge"
+
+[dependencies]
+"wasmer/static-web-server" = "^1"
+
+[fs]
+public = "public"
+config = "config" # 👈🏼 Add the config directory
+
+[[command]]
+name = "script"
+module = "wasmer/static-web-server-async:webserver"
+runner = "https://webc.org/runner/wasi"
+
+[command.annotations.wasi] # 👈🏼 Add config.toml as an argument for the webserver
+main-args = ["-w", "./config/config.toml"]
```

Comment:
Absolute paths

Reply:
changed that to use the path `/config/config.toml`

Reply:
Nope, it should be `/app/config.toml`